### ISSUE TITLE
Office documents conversion (unoconv) error running

### ISSUE THREAD

**Issue by luis100** [keeps/roda#915]
Using docker built from commit 8ec97feeb4f84b74430025983abc3f8d6a99a193
Running on [demo.docx](https://github.com/keeps/roda/files/952491/demo.docx)
With no input format selected, and create dissemination de-selected, gave the following error.

```
Command [/usr/bin/unoconv, -f, pdf, -o, /usr/local/tomcat/temp/converted1618470943582672783.pdf, /root/.roda/data/storage/aip/c14c667e-cc31-49a7-9c1a-f5a998aeb143/representations/b4b6dee2-cde2-4879-b641-32f1d16808e9/data/text/demo.docx] terminated with error code 251
```

**Comment by luis100** [comment]
But it did work on [simple.odt.zip](https://github.com/keeps/roda/files/952503/simple.odt.zip) (unzip first)

**Comment by luis100** [comment]
Further testing showed that it consistently fails on the first conversion after docker container starts, independently of the format used.

**Comment by jmaferreira** [comment]
Check if libreoffice stays running after running the command. If so, start the headless libreoffice in the boot process.

**Comment by nunovieira220** [comment]
Solved on 3a585cc0c11abc34422c4bf9513b1d0d9c3b724d
